contracts/EtherRouter.sol

```diff
@@ -48,6 +48,9 @@ contract EtherRouter is DSAuth {
 
     // Make the call
     assembly {
+      let size := extcodesize(destination)
+      if eq(size, 0) { revert(0,0) }
```

Comment:
👍 
Going though this contract I feel we may want to reword the large comment above here to make it clear which choice we made in the design ( i.e. option 2). I remember hearing at least one contributor being confused by this comment and unclear on what option we went with.